snapcraft.yaml

```diff
@@ -180,6 +180,9 @@ parts:
       - xdg-user-dirs
       - ibus-gtk
       - libibus-1.0-5
+    override-build: |
+      snapcraftctl build
+      mkdir -pv $SNAPCRAFT_PART_INSTALL/data-dir
```

Comment:
This is the desktop-gtk2 part, which won't use gtk3 theming.
According to https://forum.snapcraft.io/t/improving-theme-support-for-gtk-2-apps/7693 (if the plan still holds), gtk2 themes won't be mounted at $SNAP/data-dir/themes, so this isn't needed?

Reply:
My understanding is the theme data will be mounted there, the engines separately.

Reply:
That's right, forgive my mistake, I hadn't read the update James posted at https://forum.snapcraft.io/t/improving-theme-support-for-gtk-2-apps/7693/5.